tests/model_geometry_test.js

```diff
@@ -0,0 +1,486 @@
+'use strict';
+
+require('./helper');
+
+const { expect } = require('chai'),
+  { DataTypes } = require('../source'),
+  dialect = 'postgres',
+  semver = require('semver');
+
+const version_helper = require('../source/version_helper.js')
+const { makeTestSequelizeInstance } = require('./helper.js')
+
+function shouldSkip(test, isCRDBVersion21_2Plus) {
```

Comment:
nit: rename param to `shouldSkip` or `skipFlag` or something?

Reply:
Wait oops I re-did it since the other approach was causing CI to hang for some reason (wasn't sure why or if it was even the reason)

Also updated how were looking for the version, passing a flag for the test config instead of querying for the version.